pkg/test/configSchema.go

```diff
@@ -0,0 +1,76 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/conductorone/baton-sdk/pkg/field"
+	"github.com/conductorone/baton-sdk/pkg/ustrings"
+	"github.com/spf13/viper"
+)
+
+func MakeViper(input map[string]string) *viper.Viper {
+	output := viper.New()
+	for key, value := range input {
+		output.Set(key, value)
+	}
+	return output
+}
+
+// AssertOutcome - call a validation function and assert the result.
+func AssertOutcome(
+	t *testing.T,
+	function func() error,
+	expectedSuccess bool,
+) {
+	err := function()
+	if err != nil {
+		if expectedSuccess {
+			t.Fatal("expected function to succeed, but", err.Error())
+		}
+	} else {
+		if !expectedSuccess {
+			t.Fatal("expected function to fail, but it succeeded")
+		}
+	}
+}
+
+type TestCase = struct {
+	expression string
```

Comment:
It might be easier for the connector tests if the input was a `map[string]string` instead.

Reply:
did you change that?

Reply:
Just made the change now. Test cases can either pass a `map[string]string` or an arbitrary `string` and a parsing function.